src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -888,7 +988,472 @@ - (void)test005_DoBDXTransferAllowUpdateRequest
     // to _any_ of the above expectations.
     [self waitForExpectations:@[ announceResponseExpectation ] timeout:kTimeoutInSeconds];
 }
-#endif // ENABLE_TEST_005
+
+- (void)test007_DoBDXTransferWithTwoOTARequesters
+{
+    // In this test we do the following:
+    //
+    // 1) Check if the ota image files and raw image files required for this test exist for both OTA requestors
+    // 2) Advertise ourselves to the first OTA requestor.
+    // 3) When device1 queries for an image, pass the image path for the ota file generated beforehand as a pre-requisite for first
+    // OTA requestor 4) When device1 tries to start a bdx transfer, respond with success for the first OTA requestor
```

Comment:
Please fix this comment up...  Something went very wrong with the wrapping here.